Jenkinsfile

```diff
@@ -22,11 +22,18 @@ node {
     }
 
 
-    // stage('Test') {
-    //     tryStep "Test", {
-    //         sh "api/deploy/test/jenkins-script.sh api/deploy/test/"
-    //     }
-    // }
+    stage('Test') {
+        String PROJECT = "blackspots-unittests-${env.GIT_COMMIT}"
+
+        tryStep "unittests start", {
+            sh "docker-compose -p ${PROJECT} up --build --exit-code-from unittest unittest"
+        }
+        always {
```

Comment:
This block will never be called. Do it like this:
```
tryStep "unittests start", {
    sh "docker-compose -p ${PROJECT} up --build --exit-code-from unittest unittest"
}, {
    sh "docker-compose -p ${PROJECT} down -v || true"
}
```

That also goes for the other steps in this file

Reply:
thanks!